src/Java.Interop.Tools.Maven/Models/Project.Partial.cs

```diff
@@ -0,0 +1,62 @@
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Xml;
+using System.Xml.Serialization;
+using Java.Interop.Tools.Maven.Extensions;
+
+namespace Java.Interop.Tools.Maven.Models;
+
+public partial class Project
+{
+	public static Project Parse (Stream stream)
+	{
+		Project? result = null;
+
+		var serializer = new XmlSerializer (typeof (Project));
+
+		using (var sr = new StreamReader (stream))
```

Comment:
Why does this wrap a `Stream` in a `StreamReader`?  The [`XmlTextReader(Stream)` constructor exists](https://learn.microsoft.com/en-us/dotnet/api/system.xml.xmltextreader.-ctor?view=net-8.0#system-xml-xmltextreader-ctor(system-io-stream)).

Reply:
Fixed.